spec/build_spec.rb

```diff
@@ -48,7 +48,9 @@ def assets
 
 class DummyCommit2
   attr_reader :author
-  def initialize(published_date)
+  attr_reader :message
+  def initialize(published_date, message)
+    @message = message
```

Comment:
In order to mimic the way Octokit passes back the commit message I had to put the message member variable deep inside this extra layer.